features/step_definitions/et3_employers_contract_claim_steps.rb

```diff
@@ -0,0 +1,19 @@
+Given(/^I am on the ET3 employers contract claim page$/) do
+  @respondent = FactoryBot.create(:et3_respondent)
+  @claimant = FactoryBot.create(:et3_claimant)
+  @representative = FactoryBot.create_list(:representative, 1, :et3_information)
```

Comment:
We might get in a mess with this as sometimes @respondent is a list and sometimes @respondent is a single object ? same with others as well.  If we then write steps expecting it to be a list but a different setup is run so it is single.  Maybe we should make sure we pluralise anything that is a list ?

Reply:
@sivvoy can you give me your opinion on this?